### PR TITLE
fix(videoplayer): change partnerid format for metadata api call

### PR DESCRIPTION
**Pull request by annawen1**
### Related Ticket(s)

Video Component light box media viewer does not work for many Kaltura Videos #2303

### Description

Metadata call for video fails when using production kaltura video id. 

The partnerid needs to be passed in with a `_` in front. The description for the video comes back with html tags which we need to remove 

EXPECTED:
Screen Shot 2020-05-14 at 8 31 01 AM

Screen Shot 2020-05-14 at 8 35 15 AM

### Changelog

**New**

- add functionality in `markDowntoHtml` util to return just the string

**Changed**

- use `markDowntoHtml` in LightboxMediaViewer and VideoPlayer for the video description

### CODE REVIEW

File: packages/utilities/src/utilities/markdownToHtml/markdownToHtml.js [hunk 1]
```diff
@@ -58,10 +58,13 @@ function markdownToHtml(
     bold = true,
     useCarbonClasses = true,
     allowHtml = false,
+    cleanString = false,
+    textOnly = false,
```
Comment (jeffchew) [inline packages/utilities/src/utilities/markdownToHtml/markdownToHtml.js:62]: @annawen1 can you add JSDocs for this?

File: packages/utilities/src/utilities/markdownToHtml/markdownToHtml.js [hunk 2]
```diff
@@ -87,7 +90,7 @@ function markdownToHtml(
       });
     }
 
-    paraList += `<p>${para}</p>`;
+    paraList += textOnly ? para : `<p>${para}</p>`;
```
Comment (jeffchew) [inline packages/utilities/src/utilities/markdownToHtml/markdownToHtml.js:93]: @annawen1 `textOnly`, I'm wondering, if this is the right name for this. This is pretty much removing the paragraph functionality, right? Maybe we can call it `createParagraphs` and have it be default as true?

Comment (annawen1) [reply packages/utilities/src/utilities/markdownToHtml/markdownToHtml.js:93]: ok!

File: packages/react/src/components/VideoPlayer/VideoPlayer.js [hunk 3]
```diff
@@ -55,7 +58,12 @@ const VideoPlayer = ({ inverse, showCaption, videoId, customClassName }) => {
       </div>
       {showCaption && (
         <div className={`${prefix}--video-player__video-caption`}>
-          {videoData.name} {videoDuration}
+          {videoData.description &&
+            markdownToHtml(videoData.description, {
+              textOnly: true,
```
Comment (jeffchew) [inline packages/react/src/components/VideoPlayer/VideoPlayer.js:63]: @annawen1 you will need to update this to `createParagraphs: false`

Comment (annawen1) [reply packages/react/src/components/VideoPlayer/VideoPlayer.js:63]: ack right

File: packages/react/src/components/VideoPlayer/VideoPlayer.js [hunk 4]
```diff
@@ -55,7 +61,7 @@ const VideoPlayer = ({ inverse, showCaption, videoId, customClassName }) => {
       </div>
       {showCaption && (
         <div className={`${prefix}--video-player__video-caption`}>
-          {videoData.name} {videoDuration}
+          {videoDesc} {videoDuration}
```
Comment (kennylam) [inline packages/react/src/components/VideoPlayer/VideoPlayer.js:64]: @annawen1 The caption should actually use `videoData.name`. This was fixed in #2363

Comment (annawen1) [reply packages/react/src/components/VideoPlayer/VideoPlayer.js:64]: oh whoops - didn't notice that when i was dealing with the merge conflicts, my bad!

**Pull request merged**